airbyte-integrations/connectors/source-dv-360/source_dv_360/streams.py

```diff
@@ -0,0 +1,349 @@
+
+from abc import ABC
+from typing import Any, Iterable, Mapping, Optional, Tuple, MutableMapping
+from xmlrpc.client import Boolean
+from airbyte_cdk.sources.streams import Stream
+import json
+from typing import  Mapping, Any, List
+import pendulum
+from  google.oauth2.credentials import Credentials
+from googleapiclient.discovery import build
+import io
+import csv
+import requests
+from .fields import API_REPORT_BUILDER_MAPPING, sanitize
+from airbyte_cdk.models import SyncMode
+
+# Mapping between the schema names and the report types in the report builder
+REPORT_TYPE_MAPPING = {
+    "audience_composition": "TYPE_AUDIENCE_COMPOSITION",
+    "reach": "TYPE_REACH_AND_FREQUENCY",
+    "floodlight": "FLOODLIGHT",
+    "standard": "TYPE_GENERAL",
+    "unique_reach_audience": "TYPE_REACH_AUDIENCE"
+}
+
+def chunk_date_range(
+  start_date: str,
+  field: str,
+  end_date: str = None,
+  range_days: int = None,
+) -> Iterable[Mapping[str, any]]:
+  """

```

Comment:
please run `./gradlew format` and commit only your files this is not the default formating style.

Reply:
@marcosmarxm any hints?